### PR TITLE
Chap7 edits

### PR DESCRIPTION
**Pull request by CarlyLanglois**

### CODE REVIEW

**Review by jimflores5 — CHANGES_REQUESTED** [review]
Carly, good edits!  I think this time there's only one comment that doesn't involve small typo fixes or minor word changes.

File: src/chapters/strings/more-on-brackets.rst [hunk 1]
```diff
@@ -108,19 +106,19 @@ Check Your Understanding
 
 .. admonition:: Question
 
-   Given ``language = 'Python'``, what does ``language[1:4]`` return?
+   Given ``language = 'Python'``, what does ``language[1:5]`` return?
```
[the remark this answers is not in the document]
Comment (CarlyLanglois) [reply src/chapters/strings/more-on-brackets.rst:109]: ytho is some well known meme text so its a fun easter egg

File: src/chapters/strings/string-conditionals.rst [hunk 2]
```diff
@@ -88,17 +87,23 @@ We can use the comparison operators ``==, !=, <, >, <=, >=`` on strings.
      character to the same case (e.g.
      ``your_pet[0].lower() == other_pet[0].lower()``).
 
-#. Line 7 evaluates if ``'dog'`` comes earlier in the alphabet than ``'cat'``.
-   A string that comes earlier is considered *less than* a string that comes
+#. Line 7 evaluates if ``'dog'`` is less than ``'cat'``. This comparison is a bit more complicated.
+   What is really being compared is each character's Unicode value. All characters, even letters,
+   have Unicode values. The Unicode value of a letter corresponds with its alphabetical order. 
+   So a string that comes earlier in the alphabet is considered *less than* a string that comes
```
[the remark this answers is not in the document]
Comment (CarlyLanglois) [reply src/chapters/strings/string-conditionals.rst:93]: referring to Unicode here doesnt do harm in my eyes. this explanation doesnt ask them to get familiar with unicode and use it, but rather to know that this is the system responsible for the result of the comparison

File: src/chapters/strings/string-immutability.rst [hunk 3]
```diff
@@ -70,8 +70,8 @@ effect:
 
 .. admonition:: Note
 
-   Reassigning the value of a variable might LOOK like we are changing the
-   string, but we are NOT. Instead, we just replace the old, unchanged string
+   Reassigning the value of a variable might look like we are changing the
+   string, but we are not. Instead, we just replace the old, unchanged string
```
[the remark this answers is not in the document]
Comment (CarlyLanglois) [reply src/chapters/strings/string-immutability.rst:74]: there's a not in all caps in the code block just below this. like exclamation points, too many too frequently dilutes the impact

File: src/chapters/strings/template-literals.rst [hunk 4]
```diff
@@ -35,14 +35,14 @@ depends on multiple variables. Also, concatenation usually requires several
...
+------------------------
 
-   Beautiful is better than ugly,
+	Elegance trumps mess,
 
-   Simple is better than complex,
+	Simplicity beats complex,
 
-   Readability counts.
+	Make it readable.
```
[the remark this answers is not in the document]
Comment (CarlyLanglois) [reply src/chapters/strings/template-literals.rst]: i honestly thought you wrote a little poem here. The principles are good but i don't think we need to promote someone's dissociated distillation of a religious tenet, even if it is widely accepted within the python community. I just removed the section

File: src/chapters/strings/more-on-brackets.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/strings/more-on-brackets.rst:109]: Ah.  I'll have to Google that.

File: src/chapters/strings/string-immutability.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/strings/string-immutability.rst:74]: OK.  I see.

File: src/chapters/strings/string-conditionals.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/strings/string-conditionals.rst:93]: OK.  However, we can make this more clear if we explicitly state that a Unicode value is a number.

"...character's Unicode value, which is a number given to all symbols, letters, etc."  

Then, in the last sentence, we can reinforce this with "...earlier in the alphabet has a lower Unicode value, so it is *less than*..."

File: src/chapters/strings/string-conditionals.rst [hunk 5]
```diff
@@ -88,17 +87,23 @@ We can use the comparison operators ``==, !=, <, >, <=, >=`` on strings.
      character to the same case (e.g.
      ``your_pet[0].lower() == other_pet[0].lower()``).
 
-#. Line 7 evaluates if ``'dog'`` comes earlier in the alphabet than ``'cat'``.
-   A string that comes earlier is considered *less than* a string that comes
+#. Line 7 evaluates if ``'dog'`` is less than ``'cat'``. This comparison is a bit more complicated.
+   What is really being compared is each character's :ref:`Unicode <unicode>` value. All characters, even letters,
+   have Unicode values. The Unicode value of a letter corresponds with its alphabetical order. 
+   So a string that comes earlier in the alphabet is considered *less than* a string that comes
```
Comment (jimflores5) [inline src/chapters/strings/string-conditionals.rst:93]: I'm still struggling with this paragraph.

I see the new reference to the Unicode section, but the text there talks about special symbols and non-Latin characters.  Can we insert a simple mention here about Unicode relating to numerical values?

We don't have to go into anything like 0x0042 <---> 66 <---> "B", but helping the kids see that letters relate to numbers will clarify how the comparisons work.

**Review by jimflores5 — CHANGES_REQUESTED** [review]
Carly, we're almost there!

File: src/chapters/strings/string-conditionals.rst (hunk 5 above)
Comment (jimflores5) [reply src/chapters/strings/string-conditionals.rst]: I just kept getting hung up on "Unicode values" not clicking in the students' brains as being comparable. Calling then numbers and giving the link to the interactive table helps.

**Pull request merged**